packages/stencil-store/src/observable-map.ts

```diff
@@ -38,49 +39,49 @@ export const createObservableMap = <T extends { [key: string]: any }>(
     },
   });
 
-  const subscribe = (subscription: StoreSubscription<T> | StoreSubscriptionObject<T>): void => {
-    subscriptions.push(toSubscription(subscription));
+  const on: OnHandler<T> = (eventName, callback) => {
+    let listeners: any[] = setListeners;
+    if (eventName === 'get') {
+      listeners = getListeners;
+    } else if (eventName === 'reset') {
+      listeners = resetListeners;
+    }
+    listeners.push(callback);
+  };
+
+  const onChange: OnChangeHandler<T> = (propName, cb) => {
+    on('set', (key, newValue) => {
+      if (key === propName) {
+        cb(newValue);
+      }
+    });
+    on('reset', () => {
+      cb(defaultState[propName]);
+    });
+  };
+
+  const use = (subscription: StoreSubscriptionObject<T>): void => {
+    if (subscription.set) {
+      on('set', subscription.set);
+    }
+    if (subscription.get) {
+      on('get', subscription.get);
+    }
+    if (subscription.reset) {
+      on('reset', subscription.reset);
+    }
   };
 
   return {
-    /**
```

Comment:
Why removing documentation?

Reply:
it was not removed, we moved to the actual types, so users can see the docs!